ses

```diff
@@ -46,133 +46,184 @@ section_header "Ceph cluster status"
 
 mkdir $LOG/ceph
 
-if [ -x /usr/bin/ceph ]; then
-    ID=
-    if ! [ -f /etc/ceph/ceph.client.admin.keyring ]; then
-        # look for available keyring file and set client id accordingly
-        ID=$(ls /etc/ceph/ceph.client.*.keyring 2>/dev/null |
-                 awk -F. '{print "--id=" $3; exit}')
-    fi
+if [ -x "$(which cephadm 2>/dev/null)" ] ; then
+    CEPH_SHELL="cephadm shell --"
+    plugin_command "cephadm ls" >> $LOG/ceph/cephadm-status 2>&1
+    plugin_command "cephadm check-host" >> $LOG/ceph/cephadm-status 2>&1
+    plugin_command "cephadm inspect-image" >> $LOG/ceph/cephadm-status 2>&1
+else
+    plugin_message "cephadm not found"
+fi
 
+# TODO: for k8s/rook support, set CEPH_SHELL appropriately here
+
+if [ -n "${CEPH_SHELL}" ]; then
```

Comment:
>  it will still fall back to just invoking ceph

but where does it fall back to the invoking ceph ? Or we don't push this to the SES6 ?
I don't see CEPH_SHELL is initialized to something default. Do we need that check at all ?

Reply:
CEPH_SHELL will be set if cephadm is present. If cephadm is not present, we won't be able to collect cluster status, but that's OK, because this change is only for SES7. Older releases live in other branches (ses6, ses5, ses4).